packages/app_ui/lib/src/dialogs/bottom_animated_dialog.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class BottomAnimatedDialog {
+  static void show(BuildContext context, {required Widget child}) {
+    showGeneralDialog(
+      barrierLabel: 'BottomAnimatedDialog',
+      barrierDismissible: true,
+      barrierColor: Colors.black.withOpacity(0.5),
+      transitionDuration: const Duration(milliseconds: 500),
```

Comment:
again, we don't want to have hardcoded value, in this case it's duration.
In `app_ui` package, Let's create separate folder called const, in this folder let's create separate file called AppUiConst.
And store const called dialogAnimationDuration there.